Framework/API/inc/MantidAPI/SpectrumInfo.h

```diff
@@ -7,6 +7,7 @@
 #include <boost/shared_ptr.hpp>
 
 #include <vector>
+#include <set>
```

Comment:
@ianbush I think this is a leftover from the method you removed in your previous commit.